plugins/webhooks/index.js

```diff
@@ -343,37 +342,24 @@ function pushEvent(pluginOptions, request, reply, parsed) {
                 return reply().code(204);
             }
 
-            return pipeline.sync()
-                // handle the PR action
-                .then(p => p.jobs.then((jobs) => {
-                    const pipelineId = p.id;
-                    const name = 'main';
-                    const i = jobs.findIndex(j => j.name === name); // get job's index
-                    const jobId = jobs[i].id;
+            const eventConfig = {
+                pipelineId: pipeline.id,
+                type: 'pipeline',
```

Comment:
Technically, you don't need to pass this in because eventFactory will default `type` to 'pipeline' if none is passed in.